fe/pom.xml

```diff
@@ -558,7 +558,6 @@ under the License.
                 <groupId>org.scala-lang</groupId>
                 <artifactId>scala-library</artifactId>
                 <version>2.12.10</version>
-                <scope>provided</scope>
```

Comment:
Will removing this affect anything else?

Reply:
It will download the scala-library which will copy to fe/lib